tailwind.config.js

```diff
@@ -167,9 +204,15 @@ module.exports = {
           '1b': '#1B2942',
           '1a': '#1A202C',
           14: '#141416',
+          '4e': '#4E5D78',
+          93: '#939EAB',
+          57: '#57657B',
+          23: '#23262F',
           11: '#11142D',
         },
-        // For red, pink and orange first two chars can overlap, so identifying by middle pair
```

Comment:
Do you think this (my) comment was redundant? I thought it may be useful

Reply:
I just think there shouldn't be unnecessary comments and codes. So I deleted it. But if it is necessary for all of us, we can just leave it.

Reply:
It is explaining exception for red colors, please restore the comment to be above the red section